composer.json

```diff
@@ -42,14 +42,16 @@
         "doctrine/orm": "^2.6",
         "friendsofphp/proxy-manager-lts": "^1.0",
         "phpunit/phpunit": "^7.5 || ^8.0 || ^9.3",
+        "psalm/plugin-phpunit": "^0.15",
```

Comment:
Do you observe fewer errors when you install this plugin? Nowadays phpunit has Psalm annotations, much like Doctrine, and the README does not make clear what benefit we get from this. Also there is a Symfony plugin, maybe that one should be installed, since Symfony does not accept Psalm annotations? It would help with the `Command` and `Controller` directories, which I see are ignored for now.

Reply:
I had some issues with that locally, but on second try looks like you are right and we don't need it